src/Commands/WaContent.php

```diff
@@ -146,14 +149,19 @@ private function format_composite_contents($waContent)
                     'Widgets::InfoBox' => 'info_box',
                 ])->get($waWidget->type, null),
             ])->merge($waWidget->properties) // merge properties
-                ->merge($waWidget->image ?? null); // merge image
+            ->merge($waWidget->image ?? null); // merge image
         })->prepend($waContent->widget_content->lead_image ? // prepend lead image
             collect([
                 'type'       => 'image',
                 'lead_image' => true
             ])->merge($waContent->widget_content->lead_image)
             : null
-        )->itemsToObject();
+        )->itemsToObject()->map(function($content){
+            if($content->type === 'image' && str_contains($content->url, 'psd')) {
```

Comment:
Could we be unlucky, that the letters `psd` occures in the url, that isn't the file ending?
Does the url always end with the file extension? Then `substr($content->url, -3) === '.psd')` would be preferrable.
Otherwise, just expand the str_contains to include the period `str_contains($content->url, '.psd')`

Reply:
`parseurl($content->url, PHP_PATH)` could also be useful in case of query strings. But again - I don't know what the URLs look like.